.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - '7'
+
+  after_success:
+  - npm run build
+before_deploy:
+  - cd dist
+deploy:
+  provider: npm
+  skip_cleanup: true
+  on:
+    branch: master
+    tags: true
+    repo: kargjonas/CJSS
```

Comment:
Is this going to go badly if the fork gets out of sync with the main repo?

Reply:
Honestly I have no idea what I am doing there. Never tried Travis before. Can you help me out with this?

Reply:
I've got Travis working on my repo (https://travis-ci.com/xsanda/CJSS), and I've added linting in #35. After that I'll have a look at pushing releases

Reply:
K - nice